src/services/emailService.js

```
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: Number(process.env.SMTP_PORT || 587),
+  secure: false,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASS,
+  },
+});
```

Transporter is created with `secure: false` and no configurable TLS options. Consider deriving `secure` from the port or an env flag and allowing `tls` options (or `requireTLS`) for stricter SMTP servers; also consider calling `transporter.verify()` at startup to fail fast on bad config.